ReactiveXComponent/ReactiveXComponent.csproj

```diff
@@ -9,7 +9,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>ReactiveXComponent</RootNamespace>
     <AssemblyName>ReactiveXComponent</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.5.1</TargetFrameworkVersion>

```

Comment:
In this pull request you only need to fix the header converter for RabbitMq. Migrating to a newer version of RabbitMq client will be done in a separate pull request. We will be upgrading to the latest release version (4.1.1)